src/App.js

```diff
@@ -1,6 +1,7 @@
 import React from 'react';
 import Header from './Components/Header.jsx';
 import Main from './Components/Main.jsx';
+import SavedLists from './Components/SavedLists.jsx';
```

Comment:
You can take this out, since it looks like it's not being used in this component